### ISSUE TITLE
user control for setting the page size

### ISSUE THREAD

**Issue by stealthrabbi** [cloudeep/vuejs-paginate-next#7]
I didn't see anything in the readme, but is there a control for letting the user select the page size? I'm thinking a dropdown to choose some pre-set sizes: 5/10/20/50

**Comment by ncperng** [comment]
No, we do not have a page-size parameter. Since, vuejs-paginate-next is only a UI component, we try not to bind data information to it. Maybe you can write a separate select ui for page-size, and handle it with your api in :click-handler function?
